src/jdk.incubator.foreign/share/classes/jdk/internal/foreign/abi/x64/windows/WinVaList.java

```diff
@@ -162,6 +169,11 @@ public VaList copy() {
         return WinVaList.ofAddress(ptr);
     }
 
+    @Override
+    public VaList copy(NativeScope scope) {
+        return copy();
```

Comment:
Ok. FWIW, the Windows implementation of copy() does not need to do any allocation, because a VaList on Windows is just a pointer. But, on SysV a VaList is a struct, so we need to allocate some memory for that.